### PR TITLE
[Fix] Fixes jit tests in feature for pytorch 1.6

### PR DESCRIPTION
**Pull request by edgarriba**
### Description

This PR contains the following:
- Remove some `type: ignore` in `kornia.feature` to fix jit test issues for pytorch 1.6 introduced in https://github.com/kornia/kornia/pull/846
- Add an extra job in github action to do a quick check on different torch version to avoid post-merge errors with versions.

### Status
**Ready**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### PR Implementer
This is a small checklist for the implementation details of this PR.

If there are any questions regarding code style or other conventions check out our 
[summary](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst).

- [ ] Did you discuss the functionality or any breaking changes before ?
- [ ] **Pass all tests**: did you test in local ? `make test`
- [ ] Unittests: did you add tests for your new functionality ?
- [ ] Documentations: did you build documentation ? `make build-docs`
- [ ] Implementation: is your code well commented and follow conventions ? `make lint`
- [ ] Docstrings & Typing: has your code documentation and typing ? `make mypy`
- [ ] Update notebooks & documentation if necessary

### KorniaTeam
<details>
  <summary>KorniaTeam workflow</summary>
  
  - [ ] Assign correct label
  - [ ] Assign PR to a reviewer
  - [ ] Does this PR close an Issue? (add `closes #IssueNumber` at the bottom if 
        not already in description)

</details>

### Reviewer
<details>
  <summary>Reviewer workflow</summary>

  - [ ] Do all tests pass? (Unittests, Typing, Linting, Documentation, Environment)
  - [ ] Does the implementation follow `kornia` design conventions?
  - [ ] Is the documentation complete enough ?
  - [ ] Are the tests covering simple and corner cases ?
 
</details>

### CODE REVIEW

**Comment by edgarriba** [comment]
@ducha-aiki please, review in case I missed anything for the descriptor

File: kornia/feature/mkd.py [hunk 1]
```diff
@@ -310,7 +314,7 @@ def __init__(self,
 
         # Store precomputed embedding.
         self.register_buffer('emb', emb.unsqueeze(0))
-        self.d_emb: int = self.emb.shape[1]  # type: ignore
+        self.d_emb: int = emb.shape[0]
```
Comment (ducha-aiki) [inline kornia/feature/mkd.py:317]: Are you sure that intend to change index?

**Review by ducha-aiki — COMMENTED** [review]
LGTM except that index change

**Review by ducha-aiki — APPROVED** [review]

File: kornia/feature/mkd.py (hunk 1 above)
Comment (edgarriba) [reply kornia/feature/mkd.py:317]: there's an issue with registered buffers and typing.

Typing in pt == 1.6 doesn't seem to recognize the retrieved buffer in it's original type and assumes that could be `torch.Tensor`, `nn.Module` or `Any`.

This is why mypy doesn't recognize the variable to have a shape member. `self.emb` has shape `(1,D,...)` because of the unsqueeze from `emb`. I'm assuming here `emb.shape[0] == self.emb.shape[1]` after registering the buffer.
